src/arch/x86_64/paging/mod.rs

```diff
@@ -133,8 +133,8 @@ pub fn initialize_pagetables(
 	}
 }
 
-#[allow(dead_code)]
 /// Helper fn for debugging pagetables
+#[allow(dead_code)]
```

Comment:
```suggestion
#[cfg(test)]
```

Reply:
As far as I remember, I *presumed* that it was used for some sort of debugging but could not spot this in any tests last week IIRC. If it's actually dead code, perhaps we might want to re-implement this as a `Debug` *or* only include this function as dead code when using debug builds (but not release ones)?

Reply:
see also https://github.com/n0toose/uhyve/pull/2

Reply:
As mentioned in https://github.com/n0toose/uhyve/pull/2, I think this is a prime example of something that is intentional `dead_code` and should just be kept as is. Any of the suggestions so far only complicate the codebase just for the sake of removing the `allow`. The compiler will most likely throw out the function anyway, so I don't really see the need for any action here.

Reply:
I think this complicates the *pull request* and not the codebase - when taking a step back and considering whether this change can make sense in isolation, I think `#[cfg(test)]` says "this function is used in tests". I couldn't tell this directly earlier.

Suggestion: Let's clean up the useless allows first (as done in your other PR), and then let's build on top of that.

Reply:
Alright, then I'll merge #1020 now and leave the rebase to you :-)